test/e2e/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "akri-e2e"
+version = "0.10.10"
+description = "Akri end to end test framework"
+authors = ["Your Name <you@example.com>"]
```

Comment:
maybe just remove the authors line

```suggestion
```

Reply:
The authors field is mandatory, maybe I should just put my name here, I think we should get a policy for managing these authors fields (if I remember correctly all `Cargo.toml` mention you as author).

But definitely this placeholder is just me forgetting to fill it :innocent: